.github/workflows/test.yml

```
@@ -68,6 +68,8 @@ jobs:
           cpack -G DEB .
           dpkg -I *deb | grep "Maintainer: Foo Bar <foo@bar.local>"
           sudo -E dpkg -i *.deb
+          echo "Testing existense of libruntime_destination_dependency.a in /usr/lib"
+          test -e /usr/lib/libruntime_destination_dependency.a
```

The check is not related to the packaging process. Would it make sense to move it into a separate step instead?

```suggestion
      - name: check if we can explicitly set the runtime destination
      - run: test -e /usr/lib/libruntime_destination_dependency.a
```

---

Thank you for the review!